util/containers/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# syntax=docker/dockerfile:1.4
```

Comment:
General remark: this file duplicates quite a bit of the logic inside `build-release.sh`. Would it be possible to avoid the duplication? In particular the annoying `prefix-map` thingies below are things that one probably doesn't want to repeat. 

If we ever need to change anything about the build process, and even if we don't forget about this file, it's tricky to adjust and test this. So also from a maintenance perspective, I would prefer the Dockerfile to use `build-release.sh`. 

While I don't know a lot about Docker, I realize that there might be good reasons to do it this way. Each step here adds a new fs layer or something like that? But all the files that are created during the build (except the binary) are not part of the final container right? So it shouldn't matter? 
So yeah, what do you think?

Reply:
Using the alternative approach by mounting in the cached build output, it might be possible to reuse `build-release.sh`. But the build is relatively straightforward. Yes, the final image is very minimal, only containing the binary + some config files/libs. There should be no shell etc. Effectively: a Linux distribution containing the Alpine packages `alpine-baselayout-data`, `ca-certificates-bundle`, and `musl` + Tobira.

Reply:
As I wrote below: it sounds like a good idea to me to try out this alternative caching thingy and then using `build-release.sh`, but it's your call.